src/components/Modal/UIModal.js

```diff
@@ -50,17 +54,40 @@ const UiDataModal = ({
 
     const appNameCheck = UiData.applicationName && !/^[a-zA-Z](?:[a-zA-Z0-9_-]*[a-zA-Z0-9])?$/g.test(UiData.applicationName);
 
-    const packageNameCheck = UiData.packageName && !/^[a-zA-Z](?:[a-zA-Z0-9_.-]*[a-zA-Z0-9])?$/g.test(UiData.packageName);
-   
+    // const packageNameCheck = UiData.packageName && !/^[a-zA-Z](?:[a-zA-Z0-9_.-]*[a-zA-Z0-9])?$/g.test(UiData.packageName);
```

Comment:
remove the comments, i believe we wont be in need of any package name check of UI model in near future also.